tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_then_reset.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.clock.app import Clock
+
+class TestClockRunStopWatch(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+
+        self.clock = Clock(self.marionette)
+        self.clock.launch()
+
+    def test_click_run_stopwatch_then_reset(self):
+
+        stopwatch_view = self.clock.switch_view("stopwatch")
+
+        self.assertEqual(stopwatch_view.current_time, '00:00.00')
+
+        stopwatch_view.start()
+
+        self.assertNotEqual(stopwatch_view.current_time, '00:00.00')
+
+        stopwatch_view.pause()
+
+        self.assertNotEqual(stopwatch_view.current_time, '00:00.00')
```

Comment:
I think it would be good for a check that the pause button actually pauses too.
So something like that after 1 second or so, stopwatch_view.current_time has stayed the same.